docs/tutorials/dronecan_firmware_upgrade.rst

```diff
@@ -0,0 +1,134 @@
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/tools_bus_monitor.png
+
+The Bus Monitor tool allows you to view all DroneCAN messages being sent through the DroneCAN bus. For example, this is where you can view all of your connected 
+modules' ESC statuses. In order to start monitoring the DroneCAN bus, click the camera icon in the top left:
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/monitor_camera_button.png
+
+With modules connected and powered on, you will see ``uavcan.equipment.esc.Status`` messages appear. Click on a message to view the decoded data in the bottom left.
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/decoded_esc_status.png
+
+In order to inspect firmware update status with the Bus Monitor tool, start capturing bus activity, then trigger a firmware upgrade. You will see the USB-to-CAN node 
+transmit the ``uavcan.protocol.file.BeginFirmwareUpdate`` message, a handshake between the two nodes about the file's information through 
+the ``uavcan.protocol.file.GetInfo`` message, and then finally, the transfer of file information from the USB-to-CAN node to the Vertiq node through the 
+``uavcan.protocol.file.Read`` message.
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/monitor_read_file.png
+
+Firmware Update While Connected with a Flight Controller
+=============================================================
+An important feature of CAN is that it is a fully bussable protocol. As such, multiple DroneCAN devices can be connected to the same bus simultaneously without issue. 
+Suppose you have configured your DroneCAN hardware as recommended by `PX4's documentation <https://docs.px4.io/main/en/can/#wiring>`_.
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/px4_wiring_pic.png
+    :scale: 70%
+
+Now, you would like to update your Vertiq module firmware without dismantling your vehicle. Simply add your USB-to-CAN device to the bus chain, and connect it to the DroneCAN GUI tool. 
+In this example, we are connected to only one Vertiq module, do not have a GNSS device, and are using only one CAN output. So, our chain is as follows:
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/dronecan_upgrade_bus.png
+
+Take care to ensure that you connect all CANH signals together and all CANL signals together. If any connection is backwards, you will not be able to communicate with 
+the bus. Furthermore, all Vertiq modules include 2 sets of CANH and CANL signals. It is recommended that you use one set to interface to the flight controller and 
+the other to interface with the next device in the chain (in this case the Zubax Babel). Refer to your module's family page for more information about hardware configuration and layout.
+
+With this configuration, we expect to see the connected flight controller (in this case a PixHawk) as well as our module connected through the DroneCAN GUI:
```

Comment:
I think they don't capitalize the h, so its just Pixhawk. That's how they spell it out in their text on this page at least: https://pixhawk.org/